components/admin/FormAccess/FormAccess.tsx

```diff
@@ -7,6 +7,7 @@ import { useTranslation } from "next-i18next";
 import React, { useEffect, useState } from "react";
 import { isValidGovEmail } from "@lib/validation";
 import emailDomainList from "../../../email.domains.json";
+import server from "i18next-hmr/server";
```

Comment:
Doesn't seem this is used anywhere

Reply:
We should update our Lint rules so this throws an error on commit.